highfidelity/hifi
# Photobooth MVP - Dev tool for taking standardized product pictures

This PR contains all my work so far on the Photobooth project. It is currently at a mvp state, with basic functionalities allowing you to switch out models and take standardised pictures of the models from different camera angles.

To Test:
1) In Interface, open running script > developer > utilities > render > photobooth. Select photobooth.js and run it.
2) You should see two things: the photobooth stage appearing in front of you, and a dialog menu titled "Photobooth" popping up.
3) Test switching model: 
           - input a valid link to a FBX file of a model, then click reload button.
           - you should see the model created in centre of stage, replacing default model
           - the model might be momentarily larger than it should be, but it should automatically scale to fit
4) Test selecting camera:
          - using the drop down, select different cameras(main/left/right camera)
          - you should see your full interface view switch to the selected camera's view
          - selecting fps camera will put you back to normal view
5) Test take picture button: 
          - click the "take picture" button
          - confirm that the HUD, cursor and ui overlay disappears momentarily,  and that a new picture is saved in your screen shot folder 

Future work:
- Work with Alan to stage camera angles and backdrop
- Enable lighting presets selection (zone + light entities)
